ark/type/test/objectLiteral.test.ts

```diff
@@ -125,6 +125,42 @@ $ark.object36($arkRoot.b)`)
 		const t = type({ "a?": { b: "boolean" } })
 		attest(t.infer).typed as { a?: { b: boolean } }
 	})
+	suite("optional tuple literals", () => {
+		test("tuple with inline optional", () => {
+			const z = type(["string?", "number"])
+		})
+		test("tuple with optional tuple", () => {
+			const t = type([["string", "?"], "number", "string?"])
+		})
+		test("multidefined optional", () => {
+			const t = type(["string?", "?"])
+		})
+	})
+
+	test("symbol key", () => {
+		const s = Symbol()
+		const t = type({
+			[s]: "boolean"
+		})
+	})
+	test("optional symbol", () => {
+		const s = Symbol()
+		const t = type({
+			[s]: "boolean?"
+		})
+	})
+	test("optional keys and definition reduction", () => {
```

Comment:
Glad we got these reduce tests in. I'd probably do one as a baseline then each other case individually just so we'd know immediately if one failed which one it was.